index.html

```diff
@@ -12110,13 +12122,21 @@ <h2>Context Definitions</h2>
   <p class="changed">If an <a>expanded term definition</a> has an <code>@context</code> <a>entry</a>,
     it MUST be a valid <code>context definition</code>.</p>
 
+  <p class="changed">If a <a>context</a> contains the <code>@import</code>
```

Comment:
This whole paragraph is a copy of the one line 12017... Was the intent to replace "context" by "expanded term definition"?? That would be strange as `@import` can not appear in an expanded term definition...